modules/run_autocycler_combine.nf

```diff
@@ -0,0 +1,27 @@
+process autocycler_combine {
+  tag "AUTOCYCLER COMBINE: ${barcode}"
+  container 'quay.io/biocontainers/autocycler:0.3.0--h3ab6199_0'
+  publishDir "${params.outdir}/assemblies/${barcode}_consensus", mode: 'copy'
+
+  // errorStrategy { task.exitStatus == 1 ? 'ignore' : 'terminate' } 
```

Comment:
Is this required or can we chuck it?

Reply:
Same applies to: 

* `modules/run_autocycler_compress.nf`
* `modules/run_autocycler_resolve.nf`
* `modules/run_autocycler_subsample.nf`
* `modules/run_autocycler_table.nf`